lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -145,7 +145,7 @@ def _handle_dropped_request(self, result, d, function_name):
 
     def render(self, request):
         time_in = utils.now()
-        assert self._check_headers(request), InvalidHeaderError
+        # assert self._check_headers(request), InvalidHeaderError
```

Comment:
>I don't think we should delay the electron launch due to that.

Agreed.

>I am not convinced checking headers buys us much in the way of security

It doesn't, they can be forged, but it does rule out requests that aren't forged. I suppose I think we should change the origin to `file://`, make note that double clicking a local html file can mess up your day, and start using auth as soon as possible to correct it. In addition to using auth, we should set the api keys to only be readable by the signed app and sudo as to prevent others with access to the file system from getting an authenticated session.